packages/web-components/src/components/va-progress-bar/va-progress-bar.tsx

```diff
@@ -46,7 +46,7 @@ export class VaProgressBar {
   }
 
   render() {
-    const { label = `${this.percent}% complete`, percent } = this;
+    const { label = `${this.percent.toFixed(0)}% complete`, percent } = this;
```

Comment:
Can we get a test showing that the `aria-label` gets truncated to an integer?

Also, we're using `percent` for the `aria-valuenow` attribute as well. Can @CourtneyChambers77 check to see if that is the same kind of a11y annoyance as the `aria-label` having too many decimals?

https://github.com/department-of-veterans-affairs/component-library/blob/333cf42aaddc429945fd1bd5cb439532e697246e/packages/web-components/src/components/va-progress-bar/va-progress-bar.tsx#L57

Reply:
NVDA is reading out the entire percent including decimals for `aria-valuenow`.